### ISSUE TITLE
nternalError: (1054, u"Unknown column 'root_type' in 'field list'"

### ISSUE THREAD

**Issue by akawama-zz** [frappe/erpnext#18505]
Hello,

I am new to ERPNext and have been playing around for a while.
My issue is when I open Accounts > Cost Center Tree or try to Add Child under company name in Cost Center Tree and error message pops up.

Following are my system details.
> erpnext 11.1.47
> frappe 11.1.46
> 
> Debian 9 Stretch 4.9.0-9-amd64
> Mozilla Firefox 60.8.0

This is the error message I get
```
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 61, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 21, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 56, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1036, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/accounts/utils.py", line 735, in get_children
    acc = frappe.get_list(doctype, fields=fields, filters=filters)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 1264, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(None, *args, **kwargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/db_query.py", line 93, in execute
    result = self.build_and_run()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/model/db_query.py", line 117, in build_and_run
    return frappe.db.sql(query, as_dict=not self.as_list, debug=self.debug, update=self.update)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/database.py", line 214, in sql
    self._cursor.execute(query)
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/home/frappe/frappe-bench/env/local/lib/python2.7/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
InternalError: (1054, u"Unknown column 'root_type' in 'field list'")

```
Also its picture
![Screenshot_2019-07-28_11-08-29](https://user-images.githubusercontent.com/47711383/62002756-01000780-b12b-11e9-9f1a-aa91fe9fef89.png)

**Comment by toofun666** [comment]
Can you check the .py file content and additionally see if there is any data on the relevant table prior to that action?

**Comment by nick9822** [comment]
There were issues with the script. It is fixed with #18387 and merged into `develop` branch.
Please close the issue.
